smoke-test/tests/cypress/cypress/e2e/mutations/add_users.js

```diff
@@ -20,7 +20,7 @@ describe("add_user", () => {
         cy.visit("/settings/identities/users");
         cy.waitTextVisible("Invite Users");
 
-        cy.clickOptionWithText("Invite Users")
+        cy.clickOptionWithText("Invite Users").wait(1000);
```

Comment:
Any chance we should wait until the URL matches a particular REGEX?

Or is it re-generating a token so we cannot use a regex match here?

Reply:
yeah, it re-generating a token every time, maybe we can check for length of token in REGEX...

Reply:
Tested this REGEX locally and it works. 
"/signup\?invite_token=\w{32}/"